bokeh/plotting_helpers.py

```diff
@@ -125,6 +125,7 @@ def _process_sequence_literals(glyphclass, kwargs, source):
         kwargs[var] = var
 
 def _make_glyph(glyphclass, kws, extra):
+        if extra is None: return None
```

Comment:
It's just to make these two lines simpler: 

https://github.com/bokeh/bokeh/pull/3206/files#diff-3e6325a902133d8fd2e886fe13214143R464

If there are no `selection_` or `hover_` params, then we want to set those glyphs to `None` not to copies of the main glyph.